### PR TITLE
Check for cirq version for appropriate way to call measurement key 

### PR DESCRIPTION
**Pull request by vtomole**

### CODE REVIEW

File: cirq_superstaq/service.py [hunk 1]
```diff
@@ -42,8 +42,12 @@ def counts_to_results(
             A `cirq.Result` for the given circuit and counter.
 
     """
+    measurement_key_names = (
+        list(circuit.all_measurement_keys())
+        if cirq.__version__ <= "0.13.0"
```
Comment (richrines1) [inline cirq_superstaq/service.py:47]:
```suggestion
        if cirq.__version__ < "0.13.0"
```
(I think)

Comment (vtomole) [reply cirq_superstaq/service.py:47]: Right, done.

**Review by richrines1 — DISMISSED** [review]
lgtm

**Comment by vtomole** [comment]
@richrines1 I've changed the `if` statement to be more specific string comparison-wise. Let me know if it looks good.

**Comment by richrines1** [comment]
this seems fine currently, but will eventually break with `cirq>=0.14.0`

**Comment by vtomole** [comment]
I'm fine with that. That will remind us to modify this code so it's compatible with `measurement_keys` to existing anymore: https://github.com/SupertechLabs/cirq-superstaq/issues/60

File: cirq_superstaq/service.py [hunk 2]
```diff
@@ -42,8 +42,12 @@ def counts_to_results(
             A `cirq.Result` for the given circuit and counter.
 
     """
+    measurement_key_names = (
+        list(circuit.all_measurement_keys())
+        if "0.13.0" not in cirq.__version__
```
Comment (richrines1) [inline cirq_superstaq/service.py:47]: nit:
```suggestion
        if not cirq.__version__.startswith("0.13.0")
```

Comment (vtomole) [reply cirq_superstaq/service.py]: Done.

**Review by richrines1 — APPROVED** [review]
lgtm

**Comment by vtomole** [comment]
Let's let this sit here without merging for the next couple of days. We might not need this PR once  https://github.com/quantumlib/Cirq/pull/4403#issuecomment-929640956 happens.

**Comment by smitsanghavi** [comment]
quantumlib/Cirq#4540 should fix this.

**Comment by vtomole** [comment]
Fixed. Closing.

**Pull request closed without merging**